### ISSUE TITLE
Websocket connect issue

### ISSUE THREAD

**Issue by FakePakhan** [paulov-t/SIT.Core#99]
**Describe the bug**
There is a problem with the websocket, I have 100% open ports (6969, 6970), but when my friend connecting to the raid he can not jump, shoot and other things. In the mod config for the server I specified my external ip. When I open eft in the console says something like "[WS] Player connected", but when my friend goes in, this message is missing. The problem was solved by using WireGuard (VPN). I set up a VPN server on my local network and my friend connected, then it worked and we were able to play

**SIT.Core Version:**
1.4.8550.40954

**SPT-Aki Version:**
3.5.7

**Comment by SlejmUr** [comment]
Try External IP on http.json

**Comment by Vodqc123** [comment]
I second this ... Im going to try the vpn thingy

**Comment by FakePakhan** [comment]
I ran Aki Server on Linux using Wine and everything worked, my friend was able to connect and play. But on Windows the problem is still relevant, maybe it's a firewall problem


> Try External IP on http.json

I’ll try it, thx

**Comment by SlejmUr** [comment]
Yes, then definitely a firewall problem, if you run as admin (that you should always do when hosting servers...) You got a prompt that you wanna open the process tru firewall, tick all option that comes and press Ok/Accept.

**Comment by FakePakhan** [comment]
I have always run Aki Server as admin rights, and I clicked "OK" in the firewall window

**Comment by MR-MonkeyRay** [comment]
It happened to me. Try to disable your http proxy software before open the launcher.
https://github.com/paulov-t/SIT.Core/discussions/74#discussioncomment-6056688

**Comment by FakePakhan** [comment]
> It happened to me. Try to disable your http proxy software before open the launcher.
> [#74 (comment)](https://github.com/paulov-t/SIT.Core/discussions/74#discussioncomment-6056688)

This is the first time I've heard of http proxy, how can I turn it off?

**Comment by MR-MonkeyRay** [comment]
> > It happened to me. Try to disable your http proxy software before open the launcher.
> > [#74 (comment)](https://github.com/paulov-t/SIT.Core/discussions/74#discussioncomment-6056688)
> 
> This is the first time I've heard of http proxy, how can I turn it off?

If your friend and you are using any proxy or VPN software, you need to close them and make sure all of your windows setting in `Settings - Network & internet - Proxy` are `Off`.
Also, you can try to enable BepInEx debug log of your client to see what is going on.

**Comment by FakePakhan** [comment]
We don't use VPN or any other software (before WoreGuard), we have proxy turned off in the Windows settings, but thanks for your help

**Comment by MR-MonkeyRay** [comment]
> We don't use VPN or any other software (before WoreGuard), we have proxy turned off in the Windows settings, but thanks for your help

You're welcome, mate.
We have encountered a similar problem just now. Player A joins the host without any problem. But after player B joins the host, everyone(including B) cannot jump, run and shoot. So we suggest B ALT+F4 leave and rejoin the host. Once B exits the game, everything gets back to normal. The 2nd time B rejoins the host, he stucks at the deployment screen with 0 count down, and everyone in the server cannot jump, run and shoot again. B leaves the server and changes his network. The 3rd time he rejoins the host successfully.
So I think it might be your friend or your network problem?

**Comment by FakePakhan** [comment]
> > We don't use VPN or any other software (before WoreGuard), we have proxy turned off in the Windows settings, but thanks for your help
> 
> You're welcome, mate. We have encountered a similar problem just now. Player A joins the host without any problem. But after player B joins the host, everyone(including B) cannot jump, run and shoot. So we suggest B ALT+F4 leave and rejoin the host. Once B exits the game, everything gets back to normal. The 2nd time B rejoins the host, he stucks at the deployment screen with 0 count down, and everyone in the server cannot jump, run and shoot again. B leaves the server and changes his network. The 3rd time he rejoins the host successfully. So I think it might be your friend or your network problem?

My friend also had a problem with 0 seconds before the game started and he couldn't jump either (I could), problem solved by using a VPN or starting up an Aki Server on Linux
